lib/sinatra/base.rb

```diff
@@ -1557,11 +1558,9 @@ def caller_files
         cleaned_caller(1).flatten
       end
 
-      # Like caller_files, but containing Arrays rather than strings with the
-      # first element being the file, and the second being the line.
-      def caller_locations
-        cleaned_caller 2
-      end
+      # In Ruby 2.x+ a built-in implementation is provided for caller_locations,
+      # which is used to tag templates
+      public :caller_locations
```

Comment:
Minor: Seems like this was added in Kernel in Ruby 2.0. 2.6 is our requirement in unstable, now.

https://www.rubydoc.info/stdlib/core/2.0.0/Kernel:caller_locations

Do we need to expose it like this?

(Can be checked after merging, of course.)

Reply:
Thanks, that says it more clearly than I was saying it!